World.dm

```diff
@@ -2,20 +2,19 @@
 #define REMOTE_SERVER
 world
 	view = 7
-	version = 63
-	name = "Lands of Legend - Version 0.63"
-	status = "Lands of Legend - Version 0.63"
+	version = 64
+	name = "Lands of Legend - Version 0.64"
+	status = "Lands of Legend - Version 0.64"
 	hub = "Godsring.LandsofLegend"
 	hub_password = ""
 	loop_checks = 0
-	tick_lag = 0.75
```

Comment:
Changed to 0.50 for modernizing. raises client to around 23-29fps with very little overhead seen.